homeassistant/components/dynalite/const.py

```diff
@@ -1,31 +1,53 @@
 """Constants for the Dynalite component."""
 import logging
 
+from homeassistant.const import CONF_ROOM
+
 LOGGER = logging.getLogger(__package__)
 DOMAIN = "dynalite"
 
-ENTITY_PLATFORMS = ["light", "switch"]
+ENTITY_PLATFORMS = ["light", "switch", "cover"]
 
 
 CONF_ACTIVE = "active"
-CONF_ACTIVE_INIT = "init"
-CONF_ACTIVE_OFF = "off"
-CONF_ACTIVE_ON = "on"
+ACTIVE_INIT = "init"
+ACTIVE_OFF = "off"
+ACTIVE_ON = "on"
 CONF_ALL = "ALL"
 CONF_AREA = "area"
 CONF_AUTO_DISCOVER = "autodiscover"
 CONF_BRIDGES = "bridges"
 CONF_CHANNEL = "channel"
-CONF_CHANNEL_TYPE = "type"
+CONF_CHANNEL_COVER = "channel_cover"
+CONF_CLOSE_PRESET = "close"
 CONF_DEFAULT = "default"
+CONF_DEVICE_CLASS = "class"
+CONF_DURATION = "duration"
 CONF_FADE = "fade"
-CONF_HOST = "host"
-CONF_NAME = "name"
 CONF_NO_DEFAULT = "nodefault"
-CONF_POLLTIMER = "polltimer"
-CONF_PORT = "port"
+CONF_OPEN_PRESET = "open"
+CONF_POLL_TIMER = "polltimer"
 CONF_PRESET = "preset"
+CONF_ROOM_OFF = "room_off"
+CONF_ROOM_ON = "room_on"
+CONF_STOP_PRESET = "stop"
+CONF_TEMPLATE = "template"
+CONF_TILT_TIME = "tilt"
+CONF_TIME_COVER = "time_cover"
 
 DEFAULT_CHANNEL_TYPE = "light"
+DEFAULT_COVER_CLASS = "shutter"
```

Comment:
Use the corresponding home assistant constant.

Reply:
done